spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/BoxControllerHelperService.java

```diff
@@ -194,7 +194,7 @@ public JSONObject validateBoxInput(HttpSession session, JSONObject json) {
    *          </p>
    * @return JSONObject message indicating failure or success
    */
-  public JSONObject removeTubeFromBox(HttpSession session, JSONObject json) {
+  public JSONObject removeItemFromBox(HttpSession session, JSONObject json) {
     if (!json.has("boxId") || !json.has("position")) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "position" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)